src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoader.cs

```diff
@@ -388,6 +389,12 @@ private static void LoadGenesis(ChainSpecJson chainSpecJson, ChainSpec chainSpec
             genesisHeader.ExcessBlobGas = chainSpecJson.Genesis.ExcessBlobGas;
         }
 
+        bool isEip4788Enabled = chainSpecJson.Params.Eip4788TransitionTimestamp != null && genesisHeader.Timestamp >= chainSpecJson.Params.Eip4788TransitionTimestamp;
+        if (isEip4788Enabled)
+        {
+            genesisHeader.ParentBeaconBlockRoot = chainSpecJson.Genesis.ParentBeaconBlockRoot;
```

Comment:
Should be set to 0 unconditionally.

Reply:
still 4788 has to be enabled, so Keccak.Zero will be correct here